app/src/main/java/com/keylesspalace/tusky/db/DraftDao.kt

```diff
@@ -30,6 +31,12 @@ interface DraftDao {
     @Query("SELECT * FROM DraftEntity WHERE accountId = :accountId ORDER BY id ASC")
     fun draftsPagingSource(accountId: Long): PagingSource<Int, DraftEntity>
 
+    @Query ("SELECT COUNT(*) FROM DraftEntity where accountId = :accountId and failedToSendNew=true")
+    fun draftsNeedUserAlert(accountId: Long) : LiveData<Int>
```

Comment:
COUNT() because we don't need to see the drafts, we just need to know "0, 1, or more than one?"

Reply:
By the way do we need to add an index for failedToSendNew since we are selecting / UPDATE WHERE-ing on it?